packages/cli-exec/src/exec.js

```diff
@@ -59,6 +59,8 @@ export const exec = command('exec', {
     log.warn('Percy is disabled');
   } else {
     try {
+      percy.projectType = percy.client.tokenType();
```

Comment:
make behavior same for existing users, skip `missing token` check for all when in `--debug` or `--testing` mode ( caveat automate will have asset discovery on, which is fine for now)

more discussion happening internally.